server/service/jwt.go

```diff
@@ -8,7 +8,7 @@ import (
 
 	"github.com/src-d/code-annotation/server/model"
 
-	jwt "github.com/dgrijalva/jwt-go"
+	"github.com/dgrijalva/jwt-go"
```

Comment:
I'm sure my editor will fix it back as soon as I open this file.
I'm too lazy to look for a link right now, but it's good practice to use named import if name of package != import path.
With jwt-go it's not that bad. But sometimes names are much different and it's just difficult to understand where this package came from.